milestone2.ts

```diff
@@ -164,9 +185,16 @@ let getVolunteerInfo: GetVolunteerInfo = (vol) => {
 /* these criteria, return undefined
  */
 
-let daBigTest: GetVolunteer; // Code here
+let daBigTest: GetVolunteer = (data) => {
+  const slo_data: Volunteer | undefined = data.find(volunteer => (volunteer.city === 'SLO' && volunteer.age > 40) ? volunteer : undefined);
```

Comment:
interesting, it looks like you're sort of rewriting the find function here. The find function will actually automatically return undefined if it doesn't find someone matching the condition you pass in so this  is redundant (but not wrong!).